app/assets/stylesheets/mystyles.scss

```diff
@@ -0,0 +1,66 @@
+@charset "utf-8";
```

Comment:
what are these `mystles` sheets doing? the CSS one is so huge I can't load it without things breaking down. Double check that you actually need both. 

Additionally, there are other files I see added that have zero lines? like customClasses.css?